main.js

```diff
@@ -64,4 +66,24 @@ function showIdea() {
   }
   titleText.value = "";
   bodyText.value = "";
-}
\ No newline at end of file
+  toggleSaveButton();
+}
+
+outputContainer.addEventListener('click', function(event) {
+    if (event.target.classList.contains('delete-new-idea-img')) { 
+        var idToDelete = event.target.closest('.new-idea').getAttribute('data-id');
+
```

Comment:
This line I think can be removed.

Reply:
Can you explain a bit more why this should be removed.  The if statement starts the conditional the var idToDelete is used to remove the card from the array later in the for loop.

Reply:
@butlertree Looks like I might have selected the wrong line. I meant to just select line 75. Apologies.